.github/workflows/scripts/setup-appimage.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Script that will be run both from ci.yml and docgen.yml
+# when downloading the appimage.
+mkdir -p build
+wget https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
+chmod +x nvim.appimage
+mv nvim.appimage ./build
+# AppImages require FUSE to run: https://docs.appimage.org/user-guide/troubleshooting/fuse.html#setting-up-fuse-on-ubuntu-debian-and-their-derivatives            # NOTE: afaik contaniers can't modprobe
+# and afaik docker containers can't run kernel mdoules.
+# XXX: Should we create a script somewhere... instead of this hacky workaround??
+echo "
+#!/bin/bash
+./build/nvim.appimage --appimage-extract-and-run \"\$@\"
+" > ./build/nvim
+chmod +x ./build/nvim
```

Comment:
Why do we need this part, i dont really understand it.

The current code already works perfectly fine, in github actions?!

I don't know act or whats its purpose so

Reply:
Running the appimage directly requires FUSE, which needs to install a kernel module and can't be run on containers. The --appimage-extract-and-run flag enables executing the appimage without FUSE by extracting and instantly running. I had to male a wrapper script so that `make test` starts the appimage correctly

Reply:
There are pull requests that have failed actions and logs not available due to a long time having passed. Unless the contributor has write access or atleast access to running the actions they cannot know the problem. Being able to run failed actions locally enables us to get progress done in pull requests that may have been abandoned when they were approved and the only thing that failed were lints

Reply:
Act is this: https://github.com/nektos/act. It is a program that you install on your machine, reads the workflows directory and launches docker containers to execute the actions.

Reply:
The pull request that had failed actions with expired logs is #663, I wanted to fix the lints but I couldn't see which lints didn't pass... I tried to run it locally, it hung in the yes/no prompt from apt, I fixed that, and I then took some care of the rest, know that they were up and so on.

Reply:
Just saying i never approved this PR, there is more missing than just linting that failed and thirdly. The logs where available but this job is now so old that github deleted the logs. I maintained lets say 95% of the PRs submitted to this repo since december and we not had once an issue with appimage not running. This also applies to other repositories we maintain like plenary.nvim or fzf-native, etc ... So my question again: It works perfectly fine why do we need a weird hacky workaround for an issue we dont have.

FYI this PR https://github.com/nvim-telescope/telescope.nvim/pull/980 was just submitted that tries to implement the same feature.